app/controllers/application_controller.rb

```diff
@@ -32,8 +27,4 @@ def error(status_code, exception = nil)
       render status: status_code, text: error_message
     end
   end
-
-  def set_analytics_headers
-    set_slimmer_headers(format: "smart_answer")
```

Comment:
This one: https://github.com/alphagov/smart-answers/blob/8d1b172c7f16ec627d68429286f668d13eacb98c/app/presenters/flow_content_item.rb#L16